ProcessMaker/Http/Controllers/Api/ProcessController.php

```diff
@@ -451,18 +451,25 @@ private function saveTaskNotifications($process, $request)
     public function startProcesses(Request $request)
     {
         $where = $this->getRequestFilterBy($request, ['processes.name', 'processes.description', 'category.name']);
-        $orderBy = $this->getRequestSortBy($request, 'name');
+        $orderColumns = preg_split('/\,/', $request->input('order_by', 'name'));
```

Comment:
Why are you using regular expression of one only character instead of implode? If we are not going to accept spaces (for example) an `implode` could be more performant